src/components/LandingPage/FAQ/FAQ.tsx

```diff
@@ -12,18 +12,33 @@ interface FAQDropDownProps {
 
 const FAQDropDown: React.FC<FAQDropDownProps> = (props) => {
   const { question, answer } = props
+  let [opened, setOpen] = useState(false)
+  let [mounted, setMounted] = useState(false)
+
+  useEffect(() => {
```

Comment:
why use useEffect(), can't we use do 
```js
const [mounted, setMounted] = useState(true)
```

Reply:
no js hack, if js enabled -> set mounted flag and display: none the checkbox in favor of the js state. already tested and the behavior is not the same with usestate(true)